service/models.py

```diff
@@ -77,6 +77,7 @@ class Promotion(db.Model):  # pylint: disable=too-many-instance-attributes
     modified_by = db.Column(db.Uuid, nullable=True)
     created_when = db.Column(db.DateTime, nullable=False)
     modified_when = db.Column(db.DateTime, nullable=True)
+    activate = db.Column(db.Boolean, nullable=False, default=False)
```

Comment:
Can we change to 'active' instead of 'activate' - just so it's a noun

Reply:
Sorry for the oversight, done!

Reply:
No worries, thanks man!